catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -507,4 +508,25 @@ public static int getBit(int number, int index) {
 		}
 		return 0;
 	}
+
+	public static String stringWithoutTrailingZero(String value) {
+		String returnedValue;
+		if (value.contains(".")) {
+			try {
+				String[] splitter = value.split("\\.");
+				int afterDecimalCount = splitter[1].length();
+				String pattern = "#.";
+				for (int i = 0; i < afterDecimalCount; i++) {
+					pattern = pattern.concat("#");
```

Comment:
instead of using a for loop, you could use String.format and create a string with a given number of "#", see https://stackoverflow.com/questions/13475388/generate-fixed-length-strings-filled-with-whitespaces

Reply:
no need anymore. I've found another way to do it